simplified-books-api/src/main/java/org/nypl/simplified/books/api/BookContentProtections.kt

```diff
@@ -37,6 +37,9 @@ object BookContentProtections {
             provider.isManualPassphraseEnabled = isManualPassphraseEnabled
             provider.onLcpDialogDismissed = onLCPDialogDismissed
           }
+          else -> {
+            // do nothing
```

Comment:
It's best to avoid `else` in `when` expressions. Why? It hides a future bug: At a later date, someone adds another case to the sealed class, and then the compiler can't tell you about `when` expressions that might need updating because you used `else`. If you enumerate the cases explicitly, the compiler can tell you "You've added a new case, this `when` expression is no longer exhaustive!".

Reply:
You're absolutely right!